### PR TITLE
Adapter for FlokiFi Locker

### PR DESCRIPTION
**Pull request by AlissonRS**
##### Name (to be shown on DefiLlama):
FlokiFi Locker

##### Twitter Link:
https://twitter.com/flokifi

##### List of audit links if any:
https://www.certik.com/projects/flokifi

##### Website Link:
https://flokifi.com/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://flokifi.com/logo-only.svg

##### Current TVL:
3M

##### Chain:
Ethereum, BSC, Polygon, Arbitrum, Optimism, Avax, Fantom, OKX, KCC, Cronos, Evmos, Dogechain

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
floki

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
floki-inu

##### Short Description (to be shown on DefiLlama):
FlokiFi Locker is the industry's most innovative protocol for securing digital assets. Securely lock Liquidity Pool (LP) tokens, NFTs, fungible tokens, and Multi-tokens in just a few clicks.

##### Token address and ticker if any:
https://etherscan.io/token/0xcf0c122c6b73ff809c693db761e7baebe62b6a2e
https://bscscan.com/token/0xfb5b838b6cfeedc2873ab27866079ac55363d37e

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Services (this is too generic though, best category would be "locker" but I don't see one)

##### methodology (what is being counted as tvl, how is tvl being calculated):
SDK Adapter

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/flokifi-locker exports TVL: 
        
 
 
    fantom                    3.1727525524344236
    avax                      0.42412909564409307
    arbitrum                  1.31 k
    polygon                   4.23 k
    evmos                     0
    ethereum                  2.49 M
    okexchain                 0.01931059504490891
    kcc                       0
    cronos                    0.11107715152839281
    bsc                       393.32 k
    dogechain                 123.07 k
    optimism                  4.313857052645743
    
    total                    3.01 M

**Comment by g1nt0ki** [comment]
@AlissonRS thanks for the PR, I made a minor change here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/4979

For locked protocols, we count only LPs, I have also added whitelisted assets to be counted towards tvl, the reason for this is, some locker protocols have more tokens locked there compared to circulating supply of that coin

File: projects/helper/tokenMapping.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ const nullAddress = '0x0000000000000000000000000000000000000000'
 // orbit brige: https://bridge.orbitchain.io/open/v1/api/monitor/rawTokenList
 
 const unsupportedGeckoChains = ['aptos', 'terra2', 'terra', 'kujira']
-const ibcChains = ['terra', 'terra2', 'crescent', 'osmosis', 'kujira', 'stargaze', 'juno', 'injective', 'cosmos', 'comdex', ]
-const caseSensitiveChains = [...ibcChains, 'solana', 'tezos', 'ton', 'algorand', 'aptos', 'near', 'bitcoin', 'waves', 'tron', 'litecoin', 'polkadot', 'ripple', 'elrond', 'cardano', ]
+const ibcChains = ['terra', 'terra2', 'crescent', 'osmosis', 'kujira', 'stargaze', 'juno', 'injective', 'cosmos', 'comdex',]
+const caseSensitiveChains = [...ibcChains, 'solana', 'tezos', 'ton', 'algorand', 'aptos', 'near', 'bitcoin', 'waves', 'tron', 'litecoin', 'polkadot', 'ripple', 'elrond',]
```
[the remark this answers is not in the document]
Comment (AlissonRS) [reply projects/helper/tokenMapping.js:19]: @g1nt0ki it was by mistake when resolving conflicts, sorry about that.

**Comment by g1nt0ki** [comment]
@AlissonRS sorry, it was a mistake to whtelist Floki token, your protocol is similar to launchpads like team finance/pinksale and such, for those we count only the LPs, I have fixed it here for floki to do the same thing: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5370 

We will reset the tvl chart as it is hard to refill data in some chains

**Pull request merged**